### PR TITLE
SEO-175680-extension-UG-page-with-redirects-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

**Comment by Edberg-Vasan** [comment]
rebuild

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Syncfusion NuGet Packages Overview
 
-[NuGet](https://www.nuget.org/) can be used to automatically add files and references to your Visual Studio projects. You can use the Syncfusion NuGet packages without installing the Essential Studio or platform installation to development with the Syncfusion controls. From v16.2.0.46 (2018 Volume 2 Service Pack 1) onwards, all the Syncfusion components are available as NuGet packages at [nuget.org](https://www.nuget.org/profiles/SyncfusionInc). 
+[NuGet](https://www.nuget.org) can be used to automatically add files and references to your Visual Studio projects. You can use the Syncfusion NuGet packages without installing the Essential Studio or platform installation to development with the Syncfusion controls. From v16.2.0.46 (2018 Volume 2 Service Pack 1) onwards, all the Syncfusion components are available as NuGet packages at [nuget.org](https://www.nuget.org/profiles/SyncfusionInc).
```
Comment (Yvone-Atieno) [inline Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:12]: Revert change

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ documentation: ug
...
+[NuGet](https://www.nuget.org) can be used to automatically add files and references to your Visual Studio projects. You can use the Syncfusion NuGet packages without installing the Essential Studio or platform installation to development with the Syncfusion controls. From v16.2.0.46 (2018 Volume 2 Service Pack 1) onwards, all the Syncfusion components are available as NuGet packages at [nuget.org](https://www.nuget.org/profiles/SyncfusionInc). 
 
-N> Starting from v17.1.0.32 (2018 Volume 1), Syncfusion will no longer publish NuGet packages at [nuget.syncfusion.com](https://nuget.syncfusion.com/).
+N> Starting from v17.1.0.32 (2018 Volume 1), Syncfusion will no longer publish NuGet packages at [nuget.syncfusion.com](https://nuget.syncfusion.com).
```
Comment (Yvone-Atieno) [inline Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:14]: Revert change

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md (hunk 1 above)
Comment (Edithabich) [reply Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md]: resolved

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md (hunk 2 above)
Comment (Edithabich) [reply Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md]: resolved

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The NuGet Package Manager can be used to search and install NuGet packages in th
 3.	The Syncfusion NuGet Packages are listed the available package in the source feed URL. Search and install the required packages in your application, by clicking **Install** button.
 
 
-N> The Syncfusion NuGet packages are published in public [NuGet.org](https://www.nuget.org/) from v16.2.0.46. So, If you need to Install earlier version of 16.2.0.46 Syncfusion NuGet packages, [configure Syncfusion private feed URL](https://help.syncfusion.com/extension/syncfusion-nuget-packages/nuget-packages#syncfusion-nuget-feed-url-configuration).
+N> The Syncfusion NuGet packages are published in public [NuGet.org](https://www.nuget.org) from v16.2.0.46. So, If you need to Install earlier version of 16.2.0.46 Syncfusion NuGet packages, [configure Syncfusion private feed URL](https://help.syncfusion.com/extension/syncfusion-nuget-packages/nuget-packages#syncfusion-nuget-feed-url-configuration).
```
Comment (Yvone-Atieno) [inline Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:35]: Revert the first link changes

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md [hunk 4]
```diff
@@ -275,7 +275,7 @@ Using the NuGet CLI, all the NuGet packages in the project can be updated to the
 
 You should get the private Syncfusion NuGet feed URL to install or upgrade the Syncfusion NuGet packages. To get the URL from Syncfusion website use the following steps:
 
-1. Navigate to [nuget.syncfusion.com](https://nuget.syncfusion.com/), and select the required platform tab.     
+1. Navigate to [nuget.syncfusion.com](https://nuget.syncfusion.com), and select the required platform tab.
```
Comment (Yvone-Atieno) [inline Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:278]: Revert this change

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md (hunk 3 above)
Comment (Edithabich) [reply Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:35]: resolved

File: Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md (hunk 4 above)
Comment (Edithabich) [reply Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md:278]: resolved

**Comment by Edithabich** [comment]
No changes

**Pull request closed without merging**